xcell/mappers/mapper_base.py

```diff
@@ -18,6 +18,10 @@ def _get_defaults(self, config):
         self.nmt_field = None
         self.beam = None
         self.custom_auto = False
+        # Option introduced to modify the Mode Coupling Matrix
+        # In case the map has an implicit mask applied
+        # See ACTk case for an example
+        self.mask_power = config.get('mask_power', 1)
```

Comment:
This is repeated (see line 16)

Reply:
You are right, removed!

Reply:
would it make sense to add  `self.mask_power = config.get('mask_power', 2)` to the `__init__` function of the ACTk mapper in case we forget to define it in the params file?

Reply:
Good idea! Implemented